includes/Modules/Sign_In_With_Google.php

```diff
@@ -42,6 +43,7 @@ final class Sign_In_With_Google extends Module implements Module_With_Assets, Mo
 	 * @since 1.137.0
 	 */
 	public function register() {
+		add_action( 'login_form', array( $this, 'render_signin_button' ) );
```

Comment:
We should probably the button above the form and its white block. This will look better than render the button in the login form itself. The problem is that there is no an appropriate action for it. So, we will need to create an inline script that will create and inject needed html markup into the proper location.

![image](https://github.com/user-attachments/assets/cf82747d-afd9-4d2e-8904-1e30a73274c4)

Reply:
There is actually, `login_message` gets it where we want. I've still used CSS to get it below the error messages, but if we don't mind it being above the error messages then we can remove the CSS entirely.

<img width="407" alt="Screenshot 2024-10-24 at 16 38 23" src="https://github.com/user-attachments/assets/954e4cab-cd8f-49e0-99e5-b21c839a1f1a">